dataline-integration-tests/.gitignore

```diff
@@ -0,0 +1 @@
+config/*
```

Comment:
can you use a temp directory instead of writing in the project dir?

Reply:
these are for credentials, not things written by the test (those use temp dirs)